tutorials/end_to_end/convert_params.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import warnings
+from scipy.optimize import fsolve
+from scipy.special import zetac
+from scipy.special import erf
+
+
+# Method to convert parameters from rate to LIF
```

Comment:
This can be absolved into the docstring of the function. (It is a function, not a method.)

Reply:
Addressed in 979d63f.